rpcs3/Emu/Memory/vm.cpp

```diff
@@ -119,10 +119,38 @@ namespace vm
 		_register_lock(&cpu);
 	}
 
-	atomic_t<u64>* passive_lock(const u32 addr, const u32 end)
+	atomic_t<u64>* passive_lock(u32 addr /*mutable*/, u32 end /*mutable*/)
 	{
-		static const auto test_addr = [](const u32 target, const u32 addr, const u32 end)
+		if (addr < 0x10000 + g_locations[vm::main]->size || addr >= 0xc0000000)
```

Comment:
stack memory allows reservations.

Reply:
Well, I didn't assume otherwise. I only assumed it didn't allow oddities (shared memory).